### ISSUE TITLE
analyzer_wrapper/model_analyzer.cc has typo causing C2001 error while compiling

### ISSUE THREAD

**Issue by masaton6** [tensorflow/tensorflow#57206]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Source

source

### Tensorflow Version

tf 2.9

### Custom Code

No

### OS Platform and Distribution

Windows 10

### Python version

3.10

### Bazel version

5.0.0

### GCC/Compiler version

MSVC

### CUDA/cuDNN version

8.4.1

### GPU model and memory

GeForce GTX 750 Ti

### Current Behaviour?

```shell
While building, below error pop up

tensorflow/lite/python/analyzer_wrapper/model_analyzer.cc(197): error C2001: newline in constant

This is caused by non ascii quotes in the line 197 and 198.
```

### Standalone code to reproduce the issue

```shell
This is fixed by below patch. Please review.

diff --git a/tensorflow/lite/python/analyzer_wrapper/model_analyzer.cc b/tensorflow/lite/python/analyzer_wrapper/model_analyzer.cc
index 370a4eff470..d4609faab33 100644
--- a/tensorflow/lite/python/analyzer_wrapper/model_analyzer.cc
+++ b/tensorflow/lite/python/analyzer_wrapper/model_analyzer.cc
@@ -194,8 +194,8 @@ void dump_model_signature_defs(std::stringstream& out_stream,
     return;
   }
   out_stream << kSectionSplitter;
-  out_stream << "Your TFLite model has ‘" << signatures->Length()
-             << "’ signature_def(s).\n\n";
+  out_stream << "Your TFLite model has '" << signatures->Length()
+             << "' signature_def(s).\n\n";
   for (int i = 0; i < signatures->Length(); ++i) {
     auto* signature_def = signatures->Get(i);
     out_stream << "Signature#" << i << " key: '"
```

### Relevant log output

_No response_</details>

**Comment by mohantym** [comment]
@masaton6 !
Feel free to raise a PR . 
Thank you!

**Comment by masaton6** [comment]
Hi @mohantym ,
I have never raised PR before. Will find out how to then try.

Thank you!

**Comment by mohantym** [comment]
@masaton6 !

I do see the above change in[ line 244 ](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/python/analyzer_wrapper/model_analyzer.cc#:~:text=out_stream%20%3C%3C%20kSectionSplitter%3B,n%5Cn%22%3B) and 245 though in the master branch. 

Could you confirm from your end.

Thank you!

**Comment by masaton6** [comment]
Hi @mohantym ,

Thank you for your check. I confirmed this is already fixed with commit #2a7ad5e .

**Comment by mohantym** [comment]
Ok @masaton6 !

Thanks for the update.  Moving this to resolved status as it has fixed in the nightly version.

Thank you!
